packages/core/src/actions/index.ts

```diff
@@ -56,6 +59,20 @@ export const init = (
       uischema
     });
 
+export const registerDefaultData = (
+    schemaPath: string,
+    data: any
```

Comment:
wouldn't it be better to have a function here, which gets the schema default (if available) and returns a new value to use?
As of know I can only provide exactly one static value for a path, with a function I could make this value dynamic based on whatever information available to me.

Reply:
Yes, true, but that would introduce additional functions into the store's state, which we should avoid IMHO (although I know that we do the same with testers already).